airqo-platform/AirQo-frontend
# Location registry frontend

- What does this PR do?
This PR created a UI for the location registry component of Airqo Netmanager.
- What JIRA task is related to this PR?
The link to the JIRA board task is [here](JIRA CARD URL).
[1](https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?selectedIssue=NET-11)
[2](https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?selectedIssue=NET-23)
[3](https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?selectedIssue=NET-22)
[4](https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?selectedIssue=NET-24)
- How do I test out this PR?
Git clone this repo `cd AirQo-frontend/locate` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run local`.
- Which end points should are ready for testing?
None
